samples/CppArExample/CppArExample.cpp

```diff
@@ -361,8 +400,28 @@ void CppArExample::createSurfaceWithElevation()
   m_scene->setBaseSurface(baseSurface);
 }
 
+// Get or create graphic overlay
+Esri::ArcGISRuntime::GraphicsOverlay* CppArExample::getOrCreateGraphicsOverlay() const
```

Comment:
You don't need the full namespace here.
```suggestion
GraphicsOverlay* CppArExample::getOrCreateGraphicsOverlay() const
```